lib/airrecord/table.rb

```diff
@@ -197,6 +197,10 @@ def serializable_fields(fields = self.fields)
       }]
     end
 
+    def ==(other)
+      serializable_fields == other.serializable_fields
```

Comment:
I'm unsure if this is the right approach, but the serializable fields should be enough to determine two records equality. But please let me know if I'm off in that thinking.